plugins/outputs/cratedb/cratedb.go

```diff
@@ -115,11 +118,19 @@ func escapeValue(val interface{}) (string, error) {
 	switch t := val.(type) {
 	case string:
 		return escapeString(t, `'`), nil
-	// We don't handle uint, uint32 and uint64 here because CrateDB doesn't
-	// seem to support unsigned types. But it seems like input plugins don't
-	// produce those types, so it's hopefully ok.
-	case int, int32, int64, float32, float64:
+	case int64, float64:
 		return fmt.Sprint(t), nil
+	case uint64:
+		if t <= uint64(MaxInt64) {
+			return strconv.FormatInt(int64(t), 10), nil
+		} else {
+			return strconv.FormatInt(MaxInt64, 10), nil
+		}
```

Comment:
So the idea here is to silently truncate large numbers because CrateDB can't store them? I generally prefer to return an error in these situations, but I think there should be at least a comment explaining the reason for this code.

Reply:
Yes, this is closely based on the behavior of the InfluxDB output but I can definitely leave a comment.

We could log an error here however in most cases the value would probably be oversized on every interval, so we would want to log only once.  We can't return an error from `Write()` because then Telegraf would retry the metric.

Reply:
OK